efficientdet/keras/train_lib.py

```diff
@@ -354,9 +365,17 @@ class EfficientDetNetTrain(efficientdet_keras.EfficientDetNet):
 
   see https://www.tensorflow.org/guide/keras/customizing_what_happens_in_fit
   """
+  def __init__(self, pattern=None, *args, **kwargs):
```

Comment:
Is there any issue for the old code that use config instead of a class member?

Reply:
Thanks for pointing it out. I have removed it.